package.json

```diff
@@ -20,7 +20,99 @@
                 "command": "carbon.show",
                 "title": "Carbon"
             }
-        ]
+        ],
+        "configuration": {
+            "type": "object",
+            "title": "Carbon configuration",
+            "properties": {
+                "carbon.autoAdjustWidth": {
+                    "type": "boolean",
+                    "default": true,
+                    "description": "Automatically adjust the width of the window."
+                },
+                "carbon.backgroundColor": {
+                    "type": "string",
+                    "default": "rgba(0,0,0,0)",
+                    "description": "RGB color to use as the background behind the window.",
+                    "pattern": "^rgba\\((([01]?\\d\\d?|2[0-4]\\d|25[0-5]),\\s*){3}([01]|(0?\\.\\d+))\\)$"
```

Comment:
This is a horrible thing, but it does work. I have a simpler version if you'd prefer to use that one.

Reply:
It is ok. It can be refactored later :)